### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by benhylau**
This pulls in a bunch of patches from @darkdrgn2k which add support for:
- Orange Pi One & Orange Pi Lite (possibly other Orange Pi boards)
- Prometheus Node Exporter (report metrics)
- Prometheus Server (collect metrics)
- Grafana (visualize metrics)

In addition:
- Installs nginx and serve an HTTP-to-IPFS gateway over Port 80
- README now lists all the services available over their respective ports, so people connecting non-cjdns devices to the WiFi AP know where to access these services

Known issues:
- Have only tested on Raspberry Pi 3
- Made a bunch of changes to Orange board support, need @darkdrgn2k to test and confirm everything still works
- Grafana dashboard import isn't pulling from data_source properly yet (working on it)

### CODE REVIEW

**Comment by garrying** [comment]
@benhylau Got a tl;dr of features and changes we should be checking?

**Comment by benhylau** [comment]
Added a summary above

File: README.md [hunk 1]
```diff
@@ -33,28 +33,34 @@ The following instructions will help you set up an encrypted mesh network on Ras
...
+| `WITH_IPFS`                     | **80**: HTTP-to-IPFS gateway at `/ipfs/HASH` | Set to `true` if you want to install [IPFS](https://ipfs.io). |
+| `WITH_PROMETHEUS_NODE_EXPORTER` | **9100**: Node Exporter UI                     | Set to `true` if you want to install [Prometheus Node Exporter](https://github.com/prometheus/node_exporter) to report network metrics. |
+| `WITH_PROMETHEUS_SERVER`        | **9090**: Prometheus Server UI                 | Set to `true` if you want to install [Prometheus Server](https://github.com/prometheus/prometheus) to collect network metrics. *Requires Prometheus Node Exporter.* |
+| `WITH_GRAFANA`                  | **3000**: Grafana UI (login: admin/admin)      | Set to `true` if you want to install [Grafana](https://grafana.com) to display network metrics. *Requires Prometheus Server.* |
+| `WITH_EXTRA_TOOLS`              | None                                           | Set to `true` if you want to install non-essential tools useful for network analysis. |
```
Comment (garrying) [inline README.md]: Might be worth listing off what gets installed from `/scripts/extra-tools/install`.

File: README.md [hunk 2]
```diff
@@ -70,6 +76,10 @@ To uninstall the services, run `./prototype-cjdns-pi/scripts/uninstall`.
 
 If you are updating, run the same uninstall script, but keep all configuration files and data directories when prompted, remove the **prototype-cjdns-pi** directory along with the **install** script, then repeat the last installation step.
 
+## Experimental Support for Orange Pi
+
+We are adding support for [Orange Pi](http://www.orangepi.org/) boards and have tested with the [Orange Pi One](https://dl.armbian.com/orangepione/nightly/) and [Orange Pi Lite](https://dl.armbian.com/orangepilite/nightly/). Instead of flashing Raspbian, start with the Armbian nightly images linked above, then follow the same installation steps as the Raspberry Pi.
```
Comment (garrying) [inline README.md]: The two links to the Armbian nightly images could be more descriptive (e.g. including `nightly` and maybe `image` as well).

File: README.md [hunk 3]
```diff
@@ -33,28 +33,34 @@ The following instructions will help you set up an encrypted mesh network on Ras
...
+| `WITH_PROMETHEUS_SERVER`        | **9090**: Prometheus Server UI                 | Set to `true` if you want to install [Prometheus Server](https://github.com/prometheus/prometheus) to collect network metrics. *Requires Prometheus Node Exporter.* |
+| `WITH_GRAFANA`                  | **3000**: Grafana UI (login: admin/admin)      | Set to `true` if you want to install [Grafana](https://grafana.com) to display network metrics. *Requires Prometheus Server.* |
+| `WITH_EXTRA_TOOLS`              | None                                           | Set to `true` if you want to install non-essential tools useful for network analysis. |
 
-    ```
-    $ wget https://raw.githubusercontent.com/tomeshnet/prototype-cjdns-pi/master/scripts/install && chmod +x install && WITH_MESH_POINT=true WITH_WIFI_AP=true WITH_IPFS=true WITH_EXTRA_TOOLS=true ./install
-    ```
+If you are connected to the WiFi AP, all HTTP services are available via **http://10.0.0.1:PORT** as well as the cjdns IPv6. To connect with the cjdns address, first note your node's fc00::/8 address from `status`, then navigate to `http://[fcaa:bbbb:cccc:dddd:eeee:0000:1111:2222]:PORT` from your browser.
```
Comment (garrying) [inline README.md]: `s/AP/Access Point`

File: README.md (hunk 2 above)
Comment (darkdrgn2k) [inline README.md]: Also Orange Pi Zero @ https://dl.armbian.com/orangepizero/nightly/

**Pull request merged**